templates/ra_guac_autoscale_public_alb.template.cfn.yaml

```
@@ -432,12 +450,108 @@ Resources:
         configSets:
           config:
             - setup
+            - !If
+              - InstallCloudWatchAgent
+              - cw-agent-install
+              - !Ref "AWS::NoValue"
             - make-guac
             - finalize
           update:
             - setup
             - make-guac
             - finalize
+        cw-agent-install:
+          commands:
+            01-install-cloudwatch-agent:
+              command: !Sub >-
+                aws s3 cp ${CloudWatchAgentUrl} /etc/cfn/scripts/amazon-cloudwatch-agent.rpm &&
+                rpm -v -U /etc/cfn/scripts/amazon-cloudwatch-agent.rpm
```

not sure why we went with rpm elsewhere. try:

```suggestion
                yum localinstall /etc/cfn/scripts/amazon-cloudwatch-agent.rpm
```